src/glass/src/app/core/modals/cephfs/cephfs-modal.component.ts

```diff
@@ -79,14 +80,14 @@ export class CephfsModalComponent implements OnInit {
       return;
     }
     const values = this.formGroup.value;
-    this.blockUI.start(TEXT('Please wait, assessing service specifications ...'));
+    this.blockUI.start(translate(TEXT('Please wait, assessing service specifications ...')));
```

Comment:
Doesn't `TEXT` which is `marker` from `ngx-translate-extract-marker` already translate the text?

Reply:
Hmmm if we don't need it here, we might need to rework 

![Screenshot_2021-04-06_13-00-22](https://user-images.githubusercontent.com/8761082/113701320-4afb0800-96d8-11eb-9474-074576fc9be8.png)

Reply:
> Doesn't `TEXT` which is `marker` from `ngx-translate-extract-marker` already translate the text?

No, it's only used for marking text to be extracted.